src/modules/developer_tools.json

```diff
@@ -7,77 +7,47 @@
     },
     "methods": [
         {
-            "name": "scan",
+            "name": "find",
             "summary": "Starts a long-running DNS-SD search for a list of developer tools that match the standard 'Firebolt Developer Tool' service type",
             "tags": [
                 {
-                    "name": "capabilities",
-                    "x-manages": [
-                        "xrn:firebolt:capability:settings:developer-mode"
-                    ]
+                    "name": "temporal-set"
```

Comment:
Don't we need/want the capability-related tag, too?

Reply:
Added capabilities back in. this API actually _uses_ a capability, the developer tools capability.
The _manage_ role is for turning things on/off, renaming them, etc. But this API is actually finding & contacting to things